test/development/app-dir/hmr-asset-prefix-full-url/asset-prefix.test.ts

```diff
@@ -0,0 +1,30 @@
+import { nextTestSetup } from 'e2e-utils'
+import { retry } from 'next-test-utils'
+
+// TODO: findPort not working anyway how
+const forcedPort = String(Math.floor(10000 + Math.random() * 90000))
```

Comment:
We can use `createNext()` directly in `beforeAll()` to allow us to use `findPort()`

Reply:
Thank you!!